packages/dds/sequence/src/sharedStringWithSequentialId.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IMergeTreeOp, Marker,
+    MergeTreeDeltaType, reservedMarkerIdKey } from "@fluidframework/merge-tree";
+import { ISequencedDocumentMessage, MessageType } from "@fluidframework/protocol-definitions";
+import { parseHandles } from "@fluidframework/shared-object-base";
+import { createIdAfterMin, createIdBeforeMax } from "./generateSequentialId";
+import { SharedString } from "./sharedString";
+
+export function sharedStringWithSequentialIdMixin(Base: typeof SharedString = SharedString): typeof SharedString {
```

Comment:
this should live in a separate package under framework

Reply:
Same comment as I posted below. We can follow conversation there